plugins/titlegiver/titlegiver.py

```diff
@@ -22,7 +22,7 @@ def __init__(self):
     @staticmethod
     def get_title_from_url(url):
         # Fetch page (no need to verfiy SSL certs for titles)
-        response = auto_requests.get(url, verify=False, headers={"User-Agent": Titlegiver.USER_AGENT})
+        response = auto_requests.get(url, verify=False, headers={"User-Agent": Titlegiver.USER_AGENT, "Accept-Language": "en_US"})
```

Comment:
Also, why isn't these "sensible default" headers just part of the auto_requests library?  I mean both the user agent and the lang.